File: PRAK205-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
#include <math.h>

int main () 
{
    int alas, tinggi, keliling, luassegitiga, sisimiring, luas;

    printf ("Masukkan sisi tegak segitiga : ");
    scanf ("%d", &tinggi);

    printf ("Masukkan sisi miring segitiga : ");
    scanf ("%d", &sisimiring);

    printf ("\n");

    alas = sqrt (sisimiring * sisimiring - tinggi * tinggi);
    printf (" Alas = %d cm\n", alas );

    printf (" Tinggi = %d\n", tinggi);

    keliling = alas + tinggi + sisimiring;
    printf (" Keliling = %d cm\n",keliling);

    luas = alas * tinggi / 2;
    printf (" Luas = %d cm^2\n", luas);

    printf ("\n\n");

    main ();
}

File: PRAK205-2210817120007-NursyahnaPuteri.py
import math
tinggi = int (input ("Masukkan sisi tegak segitiga pythagoras : ")) 
sisimiring = int (input ("Masukan  sisi miring segitigas pythagoras : "))

alas = (math.sqrt (sisimiring * sisimiring - tinggi * tinggi))
keliling = alas + tinggi + sisimiring
luas = alas * tinggi / 2

print ("Alas = {} cm". format (round (alas,)))
print ("Tinggi = {} cm ". format (round (tinggi,)))
print ("Keliling = {} cm". format (round (keliling,)))
print ("Luas = {} cm^2". format (round (luas,)))